config/renovate/renovate.json

```diff
@@ -15,12 +15,10 @@
   "autodiscover": false,
   "pruneStaleBranches": false,
   "branchConcurrentLimit": 0,
+  "containerVulnerabilityAlerts": true,
   "customEnvVariables": {
     "GOTOOLCHAIN": "auto"
   },
-  "vulnerabilityAlerts": {
```

Comment:
I can't actually get any PR to show without this. Either both `containerVulnerabilityAlerts` and `vulnerabilityAlerts` need to be `false` or `true`. But in this setup, Renovate doesn't propose any updates, despite having the information.